Added info about android roduction builds when using adb root (#210)
frida/frida-website: _i18n/en/_docs/android.md

doing this on a rooted device, you might need to *su* and run it from that
shell.

<div class="note info">
  <h5>adb on a production build</h5>

>   <p>If you get <code>adbd cannot run as root in production builds</code> after running <code>adb root</code><br>you need to add <code>su -c</code>in the beginning of each adb shell. for example: <code>adb shell "su -c chmod 755 /data/local/tmp/frida-server"</code></p>
>   

Reviewer: ```suggestion
  <p>
    If you get <code>adbd cannot run as root in production builds</code> after
    running <code>adb root</code><br>you need to prefix each shell command with
    <code>su -c</code>. For example:
    <code>adb shell "su -c chmod 755 /data/local/tmp/frida-server"</code>
  </p>
```
